aiogram/dispatcher/filters/builtin.py

```diff
@@ -66,8 +66,20 @@ def __init__(self, commands: Union[Iterable, str],
                 @dp.message_handler(commands=['myCommand'], commands_ignore_caption=False, content_types=ContentType.ANY)
                 @dp.message_handler(Command(['myCommand'], ignore_caption=False), content_types=[ContentType.TEXT, ContentType.DOCUMENT])
         """
-        if isinstance(commands, str):
+        if isinstance(commands, (str, BotCommand)):
             commands = (commands,)
+        elif isinstance(commands, Iterable):
+            if not all(map(lambda cmd: isinstance(cmd, str), commands)):
+                raise ValueError(
+                    "Command filter doesn't support {} as input. "
```

Comment:
I guess you forgot to call `.format(...)` here

Reply:
oh, in this case, it is much easier do not output a problem type. I was remove the template from string. 

I would not like add here many cicles and conditions only for error. I think it will be ugly.